plugins/azure/fix_plugin_azure/resource/security.py

```diff
@@ -118,24 +121,69 @@ class AzureSecurityAssessment(MicrosoftResource, PhantomBaseResource):
         "assessment_status": S("properties", "status") >> Bend(AzureAssessmentStatus.mapping),
         "resource_source": S("properties", "resourceDetails", "Source"),
         "resource_id": S("properties", "resourceDetails", "ResourceId"),
+        "resource_type": S("properties", "resourceDetails", "ResourceType"),
         "additional_date": S("properties", "additionalData"),
         "azurePortalUri": S("properties", "links", "azurePortalUri"),
     }
     assessment_status: Optional[AzureAssessmentStatus] = field(default=None, metadata={'description': 'The result of the assessment'})  # fmt: skip
     resource_source: Optional[str] = field(default=None, metadata={'description': 'The source of the resource that the assessment is performed on'})  # fmt: skip
     resource_id: Optional[str] = field(default=None, metadata={'description': 'The id of the resource that the assessment is performed on'})  # fmt: skip
+    resource_type: Optional[str] = field(default=None, metadata={'description': 'The resource type'})  # fmt: skip
     additional_data: Optional[Dict[str, Any]] = field(default=None, metadata={'description': 'Additional data for the assessment'})  # fmt: skip
     subscription_issue: Optional[bool] = field(default=False, metadata={'description': 'Indicates if the assessment is a subscription issue'})  # fmt: skip
 
-    def post_process(self, builder: GraphBuilder, source: Json) -> None:
-        # mark as subscription issue, when the resource id is the same as the account id
-        if (rid := self.resource_id) and (sub := self._account):
-            self.subscription_issue = rid.split("/")[-1] == sub.id
+    def parse_finding(self, source: Json) -> Finding:
+        severity_mapping = {
+            "INFORMATIONAL": Severity.info,
+            "LOW": Severity.low,
+            "MEDIUM": Severity.medium,
+            "HIGH": Severity.high,
+            "CRITICAL": Severity.critical,
+        }
+        remidiation = finding_title = self.safe_name
```

Comment:
remediation and title is the name?

Reply:
yes. example from azure rest api:
```json
    {
      "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/myRg/providers/Microsoft.Compute/virtualMachineScaleSets/vmss2/providers/Microsoft.Security/assessments/21300918-b2e3-0346-785f-c77ff57d243b",
      "name": "21300918-b2e3-0346-785f-c77ff57d243b",
      "type": "Microsoft.Security/assessments",
      "properties": {
        "resourceDetails": {
          "source": "Azure",
          "id": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/myRg/providers/Microsoft.Compute/virtualMachineScaleSets/vmss2"
        },
        "displayName": "Install endpoint protection solution on virtual machine scale sets",
        "status": {
          "code": "NotApplicable",
          "cause": "OffByPolicy",
          "description": "The effective policy for the assessment was evaluated to off - use Microsoft.Authorization/policyAssignments to turn this assessment on"
        },
        "additionalData": {
          "linkedWorkspaceId": "/subscriptions/20ff7fc3-e762-44dd-bd96-b71116dcdc23/resourceGroups/myRg/providers/Microsoft.OperationalInsights/workspaces/myLaWorkspace"
        }
      }
    }
```
for mapping `name` we take `S("properties", "displayName"),`